convert2rhel/unit_tests/__init__.py

```diff
@@ -107,6 +109,21 @@ def safe_repr(obj, short=False):
     return result[:_MAX_LENGTH] + " [truncated]..."
 
 
+def get_pytest_mark(request, mark_name):
```

Comment:
This name and docstring is confusing to me. I can't tell what it's doing until I read the code

Reply:
Would get_pytest_get_marker be better?  The pytest API uses `get_marker()` (old) and `get_closest_marker()` (new) for this functionality.
```suggestion
def get_pytest_marker(request, mark_name):
```